examples/persistent-volume-provisioning/README.md

```diff
@@ -27,13 +27,15 @@ parameters:
   type: io1
   zone: us-east-1d
   iopsPerGB: "10"
+  fsType: ext4
 ```
 
 * `type`: `io1`, `gp2`, `sc1`, `st1`. See AWS docs for details. Default: `gp2`.
 * `zone`: AWS zone. If not specified, a random zone from those where Kubernetes cluster has a node is chosen.
 * `iopsPerGB`: only for `io1` volumes. I/O operations per second per GiB. AWS volume plugin multiplies this with size of requested volume to compute IOPS of the volume and caps it at 20 000 IOPS (maximum supported by AWS, see AWS docs).
 * `encrypted`: denotes whether the EBS volume should be encrypted or not. Valid values are `true` or `false`.
 * `kmsKeyId`: optional. The full Amazon Resource Name of the key to use when encrypting the volume. If none is supplied but `encrypted` is true, a key is generated by AWS. See AWS docs for valid ARN value.
+* `fstype`: fstype that are supported by kubernetes. Default: `"ext4"`.
```

Comment:
I copied this line 1:1 from vSphere where fstype was already supported. If I change it to fsType, I'll change it there as well. What do you think?